org.scummvm.ScummVM.appdata.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<!-- Copyright 2020-2022 The ScummVM Team -->
+<!-- Copyright 2020-2023 The ScummVM Team -->
 <component type="desktop">
   <id>org.scummvm.ScummVM.desktop</id>
   <metadata_license>CC0-1.0</metadata_license>
```

Comment:
Can you please bump the license to GPLv3+? We had a change in licensing a while ago :smiley:

Reply:
https://github.com/flathub/org.scummvm.ScummVM/pull/29/files

@lotharsm Any chance you could also fix project_license in the [upstream AppData file](https://github.com/flathub/org.scummvm.ScummVM/blob/master/org.scummvm.ScummVM.appdata.xml#L6)? Thanks!

Reply:
I have submitted the upstream fix myself: https://github.com/scummvm/scummvm/pull/4718